contracts/DelayManageable.sol

```diff
@@ -21,19 +21,34 @@ contract DelayManageable is Pausable, GovManager, DelayEvents, DelayModifiers {
         address _token,
         uint256[] memory _amounts,
         uint256[] memory _startDelays,
+        uint256[] memory _cliffDelays,
         uint256[] memory _finishDelays
-    )
-        external
-        onlyOwnerOrGov
-        notZeroAddress(_token)
-        equalValue(_amounts.length, _startDelays.length)
-        equalValue(_finishDelays.length, _startDelays.length)
-        orderedArray(_amounts)
-        orderedArray(_startDelays)
-        orderedArray(_finishDelays)
-    {
-        DelayLimit[_token] = Delay(_amounts, _startDelays, _finishDelays, true);
-        emit UpdatedMinDelays(_token, _amounts, _startDelays, _finishDelays);
+    ) external onlyOwnerOrGov {
+        {
+            // Stack Too deep error fixing
```

Comment:
The "Stack too deep" error can be fixed by calling the below functions inside a modifier and then using those modifiers.

Reply:
There will be two problems.
1) If we use all modifiers instead of function calls, then we will get a contract size of more than 24kb by adding more modifiers.
2) Mixed coding style. We can't use modifiers in all cases. “Stack too deep” will show up sooner or later due to a large number of modifier calls.